CherniachukArtem/HW3/task1.py

```diff
@@ -0,0 +1,26 @@
+line_from_zen = input('Enter line from "Zen of Pyiton":\n').lower()
+
+count_better = 0
+count_never = 0
+count_is = 0
+
+for i in line_from_zen.split():
```

Comment:
good job
look at more optimal version of code

zen_Of_Python = """   """
print("In 'Zen of Python' the word \"better\" repeat", zen_Of_Python.count("better"), 'times')
print("In 'Zen of Python' the word \"never\" repeat", zen_Of_Python.count("never"), 'times')
print("In 'Zen of Python' the word \"is\" repeat", zen_Of_Python.count("is"), 'times')